lambda/delete-activity-log/delete-activity-log.ts

```diff
@@ -0,0 +1,126 @@
+import { SNSEvent } from "aws-lambda";
+import {
+  DynamoDBClient,
+  BatchWriteItemCommand,
+  WriteRequest,
+} from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, QueryCommand } from "@aws-sdk/lib-dynamodb";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { ActivityLogEntry, UserData } from "./models";
+
+const marshallOptions = {
+  convertClassInstanceToMap: true,
+};
+const translateConfig = { marshallOptions };
+
+const dynamoClient = new DynamoDBClient({});
+const dynamoDocClient = DynamoDBDocumentClient.from(
+  dynamoClient,
+  translateConfig
+);
+
+const sqsClient = new SQSClient({});
+
+export const validateUserData = (userData: UserData): UserData => {
+  if (userData.user_id) {
+    return userData;
+  }
+  throw new Error(
+    `userData did not have a user_id: ${JSON.stringify(userData)}`
+  );
+};
+
+export const getAllActivitiesoForUser = async (
+  userData: UserData
+): Promise<ActivityLogEntry[] | undefined> => {
+  const { TABLE_NAME } = process.env;
+  const command = {
+    TableName: TABLE_NAME,
+    KeyConditionExpression: "user_id = :user_id",
+    ExpressionAttributeValues: {
+      ":user_id": userData.user_id,
+    },
+    // TODO: make multiple requests when dynamo has truncated the response to 10MB
```

Comment:
Could you make a ticket on the backlog so we don't forget about this?

Reply:
Yes will add thank you for spotting this .